app/controllers/bridges/lom/tasks_controller.rb

```diff
@@ -0,0 +1,206 @@
+# frozen_string_literal: true
+
+module Bridges
+  module Lom
+    # rubocop:disable Metrics/ClassLength
+    class TasksController < ActionController::API
+      OML_SCHEMA_PATH = Rails.root.join('vendor/assets/schemas/lom_1484.12.3-2020/lom.xsd')
```

Comment:
We provide these files in a similar way as the ProFormA gem. I'm not sure if this is the best place and if this will work in production. I would just give it a try.

As already mentioned, I would rather put the schema validation in a tests suite in the future, if this is to become a real feature.

Reply:
I agree. Let's try it this way and iterate over the results once it's merged and working.